### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by Rudy-Menekam**
- setup liters
- setup webpack
- set git-flow
- Add index.html file and readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,181 @@
...
+
+I would like to thank Microverse, for my project original template design.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
```
Comment (Owusu-Desmond) [inline README.md]: - [ ] Please this comment is showing in your readme and please make sure that you remove it. :pray:

File: README.md [hunk 2]
```diff
@@ -1 +1,181 @@
...
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/default.asp">JS</a></li>
+    <li><a href="https://github.com">GIT/Github</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> **[Add CRUD method (Add,remove, update,delete)]**
```
Comment (Owusu-Desmond) [inline README.md:47]: - [ ] Please make sure that you have a list at least 3 key feature that was implemented in this project :pray:

File: README.md [hunk 3]
```diff
@@ -1 +1,181 @@
...
+git and github pages
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
```
Comment (Owusu-Desmond) [inline README.md]: - [ ] Please since you are the only contributor to this project you can singularize the `Authors` to `Author`  :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Rudy-Menekam , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @Rudy-Menekam, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did a great job making the changes requested by the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**